html/index.html.in

```diff
@@ -73,222 +72,212 @@
         <link rel="manifest" href="./manifest.json">
     </head>
     <body>
-        <span class="fa hide" id="faChecker"></span>
+        <span class="fa" id="faChecker" style="display: none;"></span>
```

Comment:
Can we avoid using inline styles? Not sure why this is needed. `.d-none`?

Reply:
You mean inline styles or this element?

Reply:
inline styles :)

Reply:
they are not needed I think. :)